packages/components/src/components/div100vh-container/div100vh-container.tsx

```diff
@@ -38,6 +38,11 @@ const Div100vhContainer = ({
         height: max_autoheight_offset ? '' : height_rule,
         maxHeight: max_autoheight_offset ? `calc(100rvh - ${max_autoheight_offset})` : '',
     };
+    React.useEffect(() => {
+        // forcing resize event to make Div100vh re-render when height_offset has changed:
+        window.dispatchEvent(new Event('resize'));
+    }, [height_offset]);
+
     if (is_bypassed) return children as JSX.Element;
```

Comment:
@farzin-deriv we don't need to insert React.Fragment in this component because of the well-known issue which was happening before with styles breaking when this component was migrated to TS earlier.

Reply:
also, this file is no longer present in the current PR.